docs/content/Getting Started Using.fsx

```diff
@@ -0,0 +1,179 @@
+In general, a `Pkg` prefix is added and dots in the package ID are replaced by underscores. But make sure to look at the [nuget.g.props](https://learn.microsoft.com/en-us/nuget/reference/msbuild-targets#restore-outputs) file in the `obj` folder for the exact string.  
+The `\analyzers\dotnet\fs` subpath is a convention analyzer authors should follow when creating their packages.
+
+At last, you can run the analyzer from the project folder:
+
+```shell
+dotnet msbuild /t:AnalyzeProject
+```
+
+## Using analyzers in a solution
+
+First, we need to add the [fsharp-analyzers](https://www.nuget.org/packages/fsharp-analyzers) dotnet tool to the tool-manifest.
+```shell
+dotnet tool install fsharp-analyzers
+```
+
+Next, add the `PackageReference` pointing to your favorite analyzers to the [Directory.Build.props](https://learn.microsoft.com/en-us/visualstudio/msbuild/customize-by-directory?view=vs-2022) file:
+
+```xml
+<ItemGroup>
+    <PackageReference Include="G-Research.FSharp.Analyzers" Version="0.1.6">
+    <PrivateAssets>all</PrivateAssets>
+    <IncludeAssets>build</IncludeAssets>
+    </PackageReference>
+</ItemGroup>
+```
+
+Add the following custom target to the [Directory.Solution.targets](https://learn.microsoft.com/en-us/visualstudio/msbuild/customize-solution-build?view=vs-2022) file to be able to invoke analysis of the whole solution:
+
+```xml
+<Project>
+
+    <ItemGroup>
+        <ProjectsToAnalyze Include="src\**\*.fsproj" />
+    </ItemGroup>
+
+    <Target Name="AnalyzeSolution">
+        <MSBuild
+                Projects="@(ProjectsToAnalyze)"
+                Targets="AnalyzeProject" />
+    </Target>
+
+</Project>
+```
+
+Finally, add the following custom target to the [Directory.Build.targets](https://learn.microsoft.com/en-us/visualstudio/msbuild/customize-by-directory?view=vs-2022) file.
+ This takes care of invoking analysis of single projects:
+```xml
+<Project>
+
+    <Target
+        Name="AnalyzeProject">
+        
+        <Message Importance="normal" Text="fsc arguments: @(FscCommandLineArgs)" />
+        <Message Importance="High" Text="Analyzing $(MSBuildProjectFile)"/>
+        <Exec
+            ContinueOnError="true"
+            Command="dotnet fsharp-analyzers --project &quot;$(MSBuildProjectFile)&quot; --analyzers-path &quot;$(PkgG-Research_FSharp_Analyzers)\analyzers\dotnet\fs&quot; --exclude-analyzer PartialAppAnalyzer --fail-on-warnings GRA-STRING-001 GRA-STRING-002 GRA-STRING-003 GRA-UNIONCASE-001 --verbose --report &quot;$(MSBuildProjectName)-analysis.sarif&quot;">
+            <Output TaskParameter="ExitCode" PropertyName="LastExitCode" />
+        </Exec>
+        <Error Condition="'$(LastExitCode)' == '-2'" Text="Problems were found $(MSBuildProjectFile)" />
+
+    </Target>
+
+</Project>
+```
+
+This is effectively the same as adding a target to each `*.proj` file which exist in subfolders.
```

Comment:
Ok, sorry, ignore my previous comment. I would move this up before showing the code snippet.
You first want to explain the train of thought and then move to the code.